Exercise_A/index.html

```diff
@@ -5,41 +5,209 @@
+                      <div class="portfolio-box-caption">
+                          <div class="portfolio-box-caption-content">
+                              <div class="project-category text-faded">
+                                  Three Thing
+                              </div>
+                              <div class="project-name">
+                                  This
+                              </div>
+                          </div>
+                      </div>
+                  </div>
+              </div>
+              <div class="col-lg-4 col-sm-6">
+                  <div class="box1">
+                      <img src="/Exercise_A/assets/images/4.jpg" class="img-responsive" alt="">
+                      <div class="portfolio-box-caption">
+                          <div class="portfolio-box-caption-content">
+                              <div class="project-category text-faded">
+                                  Four Thing
+                              </div>
+                              <div class="project-name">
+                                  Bullshit
+                              </div>
+                          </div>
+                      </div>
+                  </div>
+              </div>
+          </div>
+      </div>
+  </section>
     <!-- contact me container -->
+    <section>
+        <div class="contact">
+            <div class="contact1">My Contact information </div>
+            <hr class="hhrr">
+            <div class="contact2">Here's my phone number and email address, because hey that's part of the template and why would you not include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in from all the companies wanting my insane website design skills.</div>
+            <br>
+            <div class="contact3">Honestly this template does look really nice, though.</div>
+            <div class="contact4">
+                <div class="phone">
+                    <i class="fas fa-phone-alt fa-4x"></i>
+                    <p>+1-608-317-1888</p>
+                </div>
+                <div class="email">
+                    <i class="far fa-envelope fa-4x"></i>
+                    <p>holy@shit.com</p>
+                </div>
 
-    <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
-
-    <!-- 
-      All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
+        </div>
+    </section>
 
-      If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
-      Your custom JavaScript
-     -->
+    
+     <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    ></script>
   </body>
 </html>

```

Comment:
Great Job on the website, though I believe there can be a few improvements. 
For starters, for the This button makes the page scroll and Google That Shit seem to be switched as the Google That Shit button scrolls the page but upwards instead of downwards. Next, we can look at where the images are placed below I would suggest you use Flex
You can refer to this documentation [](https://getbootstrap.com/docs/4.0/utilities/flex/)